src/components/table/table-header/bl-table-header.ts

```diff
@@ -0,0 +1,45 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import "element-internals-polyfill";
+import { blTableTag } from "../bl-table";
+import type BlTable from "../bl-table";
+import style from "../table-header/bl-table-header.css";
+
+export const blTableHeaderTag = "bl-table-header";
+
+/**
+ * @tag bl-table-header
+ * @summary Baklava Table component
+ */
+@customElement(blTableHeaderTag)
+export default class BlTableHeader extends LitElement {
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+  /**
+   * Set table header as sticky
+   */
+  @property({ type: Boolean, reflect: true, attribute: "sticky-header" })
```

Comment:
We may not need the 'header' in the attribute name as we're directly adding the attribute to header element itself.

Reply:
We can rename sticky-header to sticky as we're already adding it to the header element.